Packs/Cybereason/Playbooks/playbook-Unisolate_Endpoint_-_Cybereason.yml

```diff
@@ -1,32 +1,50 @@
 id: Unisolate Endpoint - Cybereason
-version: -1
+version: 6
```

Comment:
Looks like **format** did not work earlier on this file. Possibly because of the issue with the environment you had.
Please format this file, then commit + push. The format will remove all these unnecessary fields that got added, and also bring back the original `fromversion` and `tests` properties of the file.

Reply:
Done